go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/projectcalico/calico
 
-go 1.22
+go 1.23.0
+
+toolchain go1.23.4
```

Comment:
> Error: go [list -e -json -compiled=true -test=false -export=false -deps=true -find=false -tags ignore_autogenerated -- ./lib/apis/...]: exit status 1: go: ../go.mod requires go >= 1.23.0 (running go 1.22.12; GOTOOLCHAIN=local)

v3.28 branch is still using golang 1.22 (ships in calico/go-build v0.91). Better to keep go 1.22.